pkgs/development/ocaml-modules/afl-persistent/default.nix

```diff
@@ -0,0 +1,39 @@
+{ stdenv, buildOcaml, fetchFromGitHub, ocaml, findlib, opam }:
+
+stdenv.mkDerivation rec {
+  pname = "afl-persistent";
+  version = "1.3";
+  name = "ocaml${ocaml.version}-${pname}-${version}";
+
+  src = fetchFromGitHub {
+    owner  = "stedolan";
+    repo   = "ocaml-${pname}";
+    rev    = "v${version}";
+    sha256 = "06yyds2vcwlfr2nd3gvyrazlijjcrd1abnvkfpkaadgwdw3qam1i";
+  };
+
+  buildInputs = [ ocaml findlib ];
+
+  # don't run tests in buildPhase
+  # don't overwrite test binary
+  postPatch = ''
+    sed -i 's/ && \.\/test$//' build.sh
+    sed -i '/^ocamlopt.*test.ml -o test$/ s/$/2/' build.sh
+    patchShebangs build.sh
+  '';
+
+  buildPhase = "./build.sh";
+  installPhase = ''
+    ${opam}/bin/opam-installer --prefix=$out --libdir=lib/ocaml/${ocaml.version}/site-lib/ ${pname}.install
```

Comment:
Please use `opaline` or `dune`. `opam` is unnecessarily bloated for this use-case (copying a few files).

Reply:
thanks good suggestion!